tests/util/test_networks.py

```diff
@@ -11,30 +12,52 @@
 assert_equal = functools.partial(th.testing.assert_close, rtol=0, atol=0)
 
 
-def test_running_norm_identity() -> None:
-    """Tests running norm starts and stays at identity function.
+NORMALIZATION_LAYERS = [networks.RunningNorm, networks.EMANorm]
 
-    Specifically, we test in evaluation mode (initializatn should not change)
-    and in training mode with already normalized data.
+
+@pytest.mark.parametrize("normalization_layer", NORMALIZATION_LAYERS)
+def test_running_norm_identity_eval(normalization_layer: Type[networks.BaseNorm]):
+    """Tests running norm starts and stays at identity function when in eval mode.
+
+    Specifically, we test in evaluation mode (initialization should not change)
+
+    Args:
+        normalization_layer: the normalization layer to be tested.
     """
-    running_norm = networks.RunningNorm(1, eps=0.0)
+    running_norm = normalization_layer(1, eps=0.0)
     x = th.Tensor([-1.0, 0.0, 7.32, 42.0])
     running_norm.eval()  # stats should not change in eval mode
-    for i in range(10):
+    for _ in range(10):
         assert_equal(running_norm.forward(x), x)
+
+
+@pytest.mark.parametrize("normalization_layer", [networks.RunningNorm])
+def test_running_norm_identity_train(normalization_layer: Type[networks.BaseNorm]):
+    """Test that the running norm will not change already normalized data.
+
+    Note that we do not test the `EMANorm` here since its parameters will
```

Comment:
Sad, can we not do a more relaxed version of this test (assert_close) for that)? Although not essential given we still have a test later checking for parameter convergence.

Reply:
I was a bit worried about reducing the tolerance on the original test so that it would pass on the new code, but it was not as bad as I thought. So I've changed it over to testing both versions now.